analyzer/src/main/kotlin/PackageManager.kt

```diff
@@ -208,7 +209,7 @@ abstract class PackageManager(
                     resolveDependencies(definitionFile)?.let {
                         result[definitionFile] = it
                     }
-                } catch (e: Exception) {
+                } catch (e: IOException) {
```

Comment:
Are we sure no other exceptions can occur? This should be very robust, to not fail the analyzer unnecessarily.

Reply:
For the current code base I'm quite confident no other exceptions can occur. We just need to ensure this will also be true for future package manager implementations...

Reply:
If you prefer, I can also suppress the *detekt* finding here.

Reply:
Yes, I think I would prefer that, especially because we also call third-party code e.g. in the Maven and Gradle implementations.

Reply:
Done.